tests/lib/rules/metadata-spacing.js

```diff
@@ -56,18 +57,44 @@ ruleTester.run('metadata-spacing', rule, {
     // @downloadURL  https://raw.githubusercontent.com/Yash-Singh1/UserScripts/main/Bottom_Padding_to_Swagger_UI/Bottom_Padding_to_Swagger_UI.user.js
     // @updateURL    https://raw.githubusercontent.com/Yash-Singh1/UserScripts/main/Bottom_Padding_to_Swagger_UI/Bottom_Padding_to_Swagger_UI.user.js
     // ==/UserScript==
+    console.log("wow")`,
+      output: `// ==UserScript==
+    // @name         Bottom Padding to Swagger UI
+    // @namespace    https://github.com/Yash-Singh1/UserScripts
+    // @version      1.3
+    // @description  Adds bottom padding to the Swagger UI
+    //
+    // @author       Yash Singh
+    // @match        https://*/*
+    // @match        http://*/*
+    // @icon         https://petstore.swagger.io/favicon-32x32.png
+    // @grant        none
+    //
+    // @homepage     https://github.com/Yash-Singh1/UserScripts/tree/main/Bottom_Padding_to_Swagger_UI#readme
+    // @homepageURL  https://github.com/Yash-Singh1/UserScripts/tree/main/Bottom_Padding_to_Swagger_UI#readme
+    // @supportURL   https://github.com/Yash-Singh1/UserScripts/issues
+    // @downloadURL  https://raw.githubusercontent.com/Yash-Singh1/UserScripts/main/Bottom_Padding_to_Swagger_UI/Bottom_Padding_to_Swagger_UI.user.js
+    // @updateURL    https://raw.githubusercontent.com/Yash-Singh1/UserScripts/main/Bottom_Padding_to_Swagger_UI/Bottom_Padding_to_Swagger_UI.user.js
+    // ==/UserScript==
+
     console.log("wow")`,
       errors: [{}]
     },
     {
       code: `// ==UserScript==
     // ==/UserScript==
+    console.log("stuff")`,
+      output: `// ==UserScript==
+    // ==/UserScript==
+
     console.log("stuff")`,
       errors: [{}]
     },
     {
       code: `// ==UserScript==
     // ==/UserScript==\r\nconsole.log("stuff")`,
+      output: `// ==UserScript==
+    // ==/UserScript==\n\r\nconsole.log("stuff")`,
```

Comment:
Could you add a few test cases where there is no code to ensure userscripts without code won't autocomplete with an extra newline?

Reply:
I though there is already one inside `valid`
```js
`// ==UserScript==
// ==/UserScript==`,
```